reactioncommerce/reaction
# refactor: use npm package for inventory plugin

Resolves https://github.com/reactioncommerce/reaction/issues/6174

This PR removes the internal `inventory` plugin, and installs the new `api-plugin-inventory` from npm as a replacement.

The code in that npm plugin is the same, so the expected outcome of this PR is seeing nothing different than how the app currently runs.

## Testing
- Start reaction, see that `inventory` is the correct version in the `systemInformation` query
- Update the inventory for a product, publish, and see the changes in the UI

Signed-off-by: Maria Paktiti <maria.paktiti@gmail.com>
